tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
@everilae: regarding
 > method is a protocol class instance and never available for users

This statement is incorrect. A number of pika functions and callbacks return the method to the user. For example, `basic_consume`'s consumer callback is explicitly called with the args `channel, method, properties, body`. Same is true for `basic_get`, etc.